ironfish-cli/src/commands/deposit.ts

```diff
@@ -167,12 +170,12 @@ The memo will contain the graffiti "${graffiti}".
 
       const transaction = result.content
       this.log(`
-Depositing ${displayIronAmountWithCurrency(IRON_TO_SEND, true)} from ${
+Depositing ${displayAmount} from ${
         transaction.fromAccountName
       }
 Transaction Hash: ${transaction.hash}
-Transaction fee: ${displayIronAmountWithCurrency(feeInIron, true)}
-
+Transaction fee: ${displayFee}
```

Comment:
using these vars that were already set is a nice cleanup

Reply:
Thank you @lwisne :)